packages/rocketchat-authorization/client/stylesheets/permissions.css

```diff
@@ -9,6 +9,10 @@
 		font-weight: bold !important;
 	}
 
+	& .section:not(.section-collapsed) {
+		inline-size: fit-content;
```

Comment:
This is the hack: I didn't get the section headline to take the full available width otherwise.
There's surely a better (all-browser-compatible) solution to that.

Reply:
not needed anymore - switched to a second table